catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/CreateOperations.java

```diff
@@ -401,6 +424,37 @@ private String buildIngestLog(CreateRequest createReq) {
     return strBuilder.toString();
   }
 
+  /** Helper method to build file name(s) string paramater: CreateStorageRequest */
+  private String buildFileNames(CreateStorageRequest createStorageRequest) {
+    StringBuilder stringBuilder = new StringBuilder();
+
+    for (int i = 0; i < createStorageRequest.getContentItems().size(); i++) {
+      if (i > 0) {
+        stringBuilder.append(", ");
+      }
+      stringBuilder.append(createStorageRequest.getContentItems().get(i).getFilename());
+    }
+    return stringBuilder.toString();
+  }
+
+  /** Helper method to build file name(s) string paramater: CreateRequest */
+  private String buildFileNames(CreateRequest createRequest) {
+    StringBuilder stringBuilder = new StringBuilder();
+    List<Metacard> metacards = createRequest.getMetacards();
+
+    for (int i = 0; i < metacards.size(); i++) {
+      Metacard card = metacards.get(i);
+      if (i > 0) {
+        stringBuilder.append(", ");
+      }
+      if (card.getTitle() != null) {
+        stringBuilder.append(card.getTitle());
+      }
+    }
+
+    return stringBuilder.toString();
+  }
```

Comment:
✏️ Same comment about the String.join() method.

Reply:
fixed